Assets/_Laz/Scripts/Lazo/Lazo.Wall.cs

```diff
@@ -17,5 +19,20 @@ private void ClearWall()
             
             _listOfPositions.Clear();
         }
+
+        private void KillOffTailEndOfLazoFrom(int closedOffPosition, LazoPosition[] closedLoop)
+        {
+            for (int i = closedOffPosition; i >= 0; i--)
+            {
+                var positionDeathTimeToLive = ((float) i / closedOffPosition) * TimeForTailToDisappear;
+                _listOfPositions[i].TimeToLive = positionDeathTimeToLive;
+            }
+            
+            //TODO - Placeholder, but it should disappear after the tail disappears
```

Comment:
what is the plan for this TODO? will it be done in another task?

Reply:
yeah, so when you close the loop, some other animations/particlefx should happen, but for now the loop should vanish.